emp.js

```diff
@@ -6,12 +6,19 @@ document.getElementById('employeeForm').addEventListener('submit', function(even
     var position = document.getElementById('position').value;
 
     // Validation to accept only alphabets, dots, and spaces for the name field
-    var regex = /^[A-Za-z .]+$/;
-    if (!regex.test(name)) {
+    var nameRegex = /^[A-Za-z .]+$/;
+    if (!nameRegex.test(name)) {
         alert('Please enter a valid name with alphabets, dots, and spaces only.');
         return;
     }
 
+    // Validation to accept only alphabets, dots, and spaces for the position field
+    var positionRegex = /^[A-Za-z .]+$/;
+    if (!positionRegex.test(position)) {
+        alert('Please enter a valid position with alphabets, dots, and spaces only.');
+        return;
+    }
+
     var result = document.getElementById('result');
     result.innerHTML = `<p>Name: ${name}</p><p>Email: ${email}</p><p>Position: ${position}</p>`;
```

Comment:
Add a validation for the email field before printing it and also it is required field.